src/video/khronos/EGL/eglext.h

```diff
@@ -812,6 +812,11 @@ EGLAPI EGLSurface EGLAPIENTRY eglCreatePlatformPixmapSurfaceEXT (EGLDisplay dpy,
 #define EGL_PLATFORM_X11_SCREEN_EXT       0x31D6
 #endif /* EGL_EXT_platform_x11 */
 
+#ifndef EGL_EXT_present_opaque
+#define EGL_EXT_present_opaque 1
+#define EGL_PRESENT_OPAQUE_EXT            0x31DF
+#endif /* EGL_EXT_protected_content */
```

Comment:
>EGL_EXT_protected_content

Only just realized this is copied from the extension below it... the Mesa patch has this too!

Reply:
Ah yes, guilty as charged :sweat_smile:

Thanks for noticing that, I'll fix it :+1:

Reply:
Btw, this `src/video/khronos/EGL/eglext.h` change was only meant to reflect the real change that will be to pull this file from upstream once the extension lands (and when I do that, other things in this file will change, starting with the date in the header, and possibly other changes that hadn't been carried over yet).

Reply:
I'll have to check and see how we do header updates here... sometimes we use the header and sometimes we define it in SDL_egl.c, I'm actually not sure what the rule is there. Ryan or Sam might know though.